client/my-sites/upgrades/checkout/style.scss

```diff
@@ -926,3 +926,10 @@
 		transform: translateZ(0) scale(1);
 	}
 }
+
+.credit-card-payment-box__progress-bar {
+	color: grey;
```

Comment:
This should be either $gray or $gray-text-min. $gray-text-min will make the text an accessible contrast.